dist/svg.js

```diff
@@ -1694,20 +1704,53 @@ SVG.FX = SVG.invent({
 
       // convert current time to position
       if(!ignoreTime) this.pos = this.timeToPos(+new Date)
+      else {
+        // If the method at was called with a position above 1, we set it back to 1
+        if(this.pos > 1) this.pos = 1
+        if(this.situation.reversed) this.pos = 1 - this.pos
+      }
 
-      if(this.pos >= 1 && (this.situation.loop === true || (typeof this.situation.loop == 'number' && --this.situation.loop))){
+      if(this.pos < 0) this.pos = 0
+      if(this.pos >= 1){
```

Comment:
I would rank simple code a lot higher than this issue. Also: allowing more is usually better. The user can decide whether to use it or not